vite.config.ts

```diff
@@ -16,5 +16,10 @@ export default defineConfig({
     rollupOptions: {
       external: ['react-router-dom']
     }
+  },
+  resolve: {
+    alias: {
+      '@': '/site'
+    }
```

Comment:
Adding this section isn't what I meant by my previous comment. I'm not sure what this even does. I mean the server proxy needs to be updated to not remove /api